test/pageobjects/page.js

```diff
@@ -0,0 +1,15 @@
+import { browser } from '@wdio/globals';
+
+/**
+ * main page object containing all methods, selectors and functionality
+ * that is shared across all page objects
+ */
+export default class Page {
+  /**
+   * Opens a sub page of the page
+   * @param path path of the sub page (e.g. /path/to/page.html)
+   */
+  open(path) {
+    return browser.url(`https://the-internet.herokuapp.com/${path}`);
```

Comment:
We can move this to the wdio.config file - https://github.com/slavian333/automation_tests/pull/2/files#diff-f297ad33212595210fbfc7c32d334f7025fe8eee14cc4179d2527c3f4543035fR99
Basically you already did it, so the method should look like 

```
open(path) {
    return browser.url(`/${path}`);
}
```

Reply:
Also was installed with wdio. Agred and modified.